src/test/java/io/stargate/sgv2/jsonapi/api/v1/UpdateManyIntegrationTest.java

```diff
@@ -217,7 +220,8 @@ public void limit() {
           .statusCode(200)
           .body("status.matchedCount", is(20))
           .body("status.modifiedCount", is(20))
-          .body("status.moreData", nullValue())
+          .body("status.moreData", is(true))
+          .body("status.nextPageState", notNullValue())
```

Comment:
I know we can't compare to specific value (as it's generated token), but maybe check it's a non-empty String (length above 0)?